catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -335,6 +340,7 @@
                             ddf.catalog.content.operation;version="2.0",
                             ddf.catalog.content.plugin;version="2.0",
                             ddf.catalog.content;version="2.0",
+                            ddf.catalog.core.versioning,
```

Comment:
Do I @kcwire ? Is there a reason? Just didn't want to do it without knowing what or why